tools/accuracy_checker/accuracy_checker/config/config_reader.py

```diff
@@ -411,6 +411,9 @@ def provide_models(launchers):
                 for launcher_entry in model['launchers']:
                     merge_dlsdk_launcher_args(arguments, launcher_entry, update_launcher_entry)
                 model['launchers'] = provide_models(model['launchers'])
+                for dataset_entry in model['datasets']:
+                    if 'subsample_size' in arguments:
+                        dataset_entry['subsample_size'] = arguments.subsample_size
```

Comment:
please make sure that it also works for pipelined  models

Reply:
please review again